### PR TITLE
Update docker.md

### PR DESCRIPTION
**Pull request by larssono**

### CODE REVIEW

File: articles/docker.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ excerpt: The Synapse Docker registry provides a space for synapse users to store
...
+Docker is a tool for creating, running and managing lightweight virtual machines, represented as versioned 'repositories.'  These virtual machines make it possible to distribute executable environments with all of the dependencies that can easily be run by others.  A Docker 'registry' is a collection of these repositories and there are a number of open registries on the web. Synapse hosts a private registry, freely available to our users. Synapse users interact with the Synapse Docker registry using the standard Docker client. As with Files and Table, Repositories are organized by project and inherit the access controls from the project.
```
Comment (kdaily) [inline articles/docker.md]: Containers are not virtual machines, nor are they represented as 'repositories'.

Runon sentence starting "A Docker 'registry' is a collection..."

Is the registry in Synapse private? Any Certified Synapse user can push to it - probably not the best word considering Synapse is not 'private' (though I note 'freely available' wording after, sounds contradictory)

'As with Files and Tables' - missing 's' on 'Table'

File: articles/docker.md [hunk 2]
```diff
@@ -6,11 +6,11 @@ excerpt: The Synapse Docker registry provides a space for synapse users to store
...
 
 
 ### Creating a new Docker image
-Lets begin by creating a custom docker image.  Users can choose to either modify an existing docker image or build a docker image from a Dockerfile.  Docker images must be tagged with 'docker.synapse.org/synapseProjectId/myreponame' to allow images to be saved. 
+Users can choose to either modify an existing docker image or build a docker image from a Dockerfile.  Docker images must be tagged with 'docker.synapse.org/synapseProjectId/myreponame' to allow images to be saved.  In this example we will pull down an existing Docker image and add it to the Synapse project, 'syn12345'.
```
Comment (kdaily) [inline articles/docker.md]: 'docker' is a proper noun (per Docker's documentation) - should be capitalized in all places in this document.

File: articles/docker.md [hunk 3]
```diff
@@ -48,16 +48,13 @@ When building a Docker image from a Dockerfile simply add a `-t` to the docker b
 docker build -t  docker.synapse.org/syn12345/my-repo path/to/dockerfile
 ```
 
-To learn more about building [docker images](https://docs.docker.com/engine/getstarted/step_four/).  
+Learn more about building [docker images](https://docs.docker.com/engine/getstarted/step_four/).
```
Comment (kdaily) [inline articles/docker.md]: Sentence fragment, this is.

File: articles/docker.md [hunk 4]
```diff
@@ -48,16 +48,13 @@ When building a Docker image from a Dockerfile simply add a `-t` to the docker b
...
 
 ### Storing Docker images in Synapse
 To store Docker images, use the `docker push` command.  To push to the Synapse Docker Registry, users must be logged into the registry:
 
 ```
-docker login docker.synapse.org
-#Username: (Input Synapse username)
-#Password: 
-#Login Succeeded 
+docker login -u <synapse username> -p <synapse password> docker.synapse.org
```
Comment (kdaily) [inline articles/docker.md]: use code syntax (```like this```)

File: articles/docker.md [hunk 5]
```diff
@@ -75,10 +72,9 @@ docker push docker.synapse.org/syn12345/my-repo
 ### Using Docker images stored in Synapse
 To access the Docker images stored in Synapse, simply use the `docker pull` command.
```
Comment (kdaily) [inline articles/docker.md]: no need for 'simply' - not simple for everyone.

File: articles/docker.md [hunk 6]
```diff
@@ -75,10 +72,9 @@ docker push docker.synapse.org/syn12345/my-repo
...
 
+{% include tip.html content="By default, if you do not specify a tag, it will attach latest as the tag.  If you specified a tag on your repository, be sure to pull the repository with the tag." %}
 ```
 docker pull docker.synapse.org/syn12345/my-repo
-#By default, if you do not specify a tag, it will attach "latest" as the tag.  If you specified a tag on your repository, be sure to pull the repository with the tag.
-docker pull docker.synapse.org/syn12345/mytestrepo:version1
 ```
 
 Docker tags can be assigned to later commits. If you want to be explicit about the version of an image then instead of referencing a tag you can reference a digest:
```
Comment (kdaily) [inline articles/docker.md]: What's a 'commit'? Not defined.

File: articles/docker.md [hunk 7]
```diff
@@ -87,4 +83,9 @@ Docker tags can be assigned to later commits. If you want to be explicit about t
 docker pull docker.synapse.org/syn12345/mytestrepo@sha256:2e36829f986351042e28242ae386913645a7b41b25844fb39b29af0bdf8dcb63
 ```
 
-where the digest for a commit is printed to the command line after a successful Docker push. The Synapse web portal displays current digests for a repository's tags on the repository's Synapse page.
\ No newline at end of file
+where the digest for a commit is printed to the command line after a successful Docker push. The Synapse web portal displays current digests for a repository's tags on the repository's Synapse page.
```
Comment (kdaily) [inline articles/docker.md]: Incomplete sentence - hard to parse?

File: articles/docker.md [hunk 8]
```diff
@@ -87,4 +83,9 @@ Docker tags can be assigned to later commits. If you want to be explicit about t
...
+
+{% include note.html content="You can add external repositories, i.e. repositories that live in other registries like DockerHub and quay.io. For these repositories there is no tight integration (Synapse doesn't contact these external registries) but it allows you to list Docker repositories that are relevant to the project but are not within Synapse.
```
Comment (kdaily) [inline articles/docker.md]: I think you mean 'e.g.' not 'i.e.'. It should also should have a trailing comma (i.e., like 'e.g.,').

Comment (kdaily) [inline articles/docker.md]: Link to [quay.io](http://quay.io).

File: articles/docker.md [hunk 9]
```diff
@@ -6,11 +6,11 @@ excerpt: The Synapse Docker registry provides a space for synapse users to store
...
+Docker is a tool for creating, running and managing lightweight virtual machines, represented as versioned 'repositories.'  These virtual machines make it possible to distribute executable environments with all of the dependencies that can easily be run by others.  A Docker 'registry' is a collection of these repositories and there are a number of open registries on the web. Synapse hosts a private registry, freely available to our users. Synapse users interact with the Synapse Docker registry using the standard Docker client. As with Files and Table, Repositories are organized by project and inherit the access controls from the project.
 
 
 ### Creating a new Docker image
```
Comment (kdaily) [inline articles/docker.md]: What's an image?

File: articles/docker.md (hunk 8 above)
Comment (brucehoff) [inline articles/docker.md]: actually I think "i.e." is correct here.  What follows is not an example but the definition of what we mean by 'external'.

**Comment by kdaily** [comment]
Indeed, you are right (i.e., I don't know how to read). Still needs a
trailing comma though!

**Comment by meredithslota** [comment]
I did the best I could to resolve the conflicts and get this out there, but it may need another read-through. Let's open a new issue for the edits, since this one was so stale.
